### ISSUE TITLE
issue with play 2.7

### ISSUE THREAD

**Issue by saloni2195** [playframework/play-spring-loader#19]
play spring loader generate issue on play 2.7. can you please tell how this works with play 2.7

**Comment by raboof** [comment]
It looks like this library still targets Play 2.6 (https://github.com/playframework/play-spring-loader/blob/master/build.sbt#L10).

Duplicate of https://discuss.lightbend.com/t/spring-integration-in-play-2-7/3787

**Comment by marcospereira** [comment]
Yes, as @raboof said, this is targeting Play 2.6. Contributions to make it work with 2.7. are welcome since it is a [community maintained](https://github.com/playframework/play-spring-loader#support) library.

Finally, what problem are you seeing? Could you create a reproducer?
